packages/syntax-core/src/SelectList/SelectList.tsx

```diff
@@ -143,7 +143,7 @@ const SelectList = ({
           size={100}
           color={errorText ? "destructive-primary" : "gray700"}
         >
-          {errorText ?? helperText}
+          {errorText ? errorText : helperText}
```

Comment:
We use this syntax instead of `errorText || helperText` to avoid triggering https://typescript-eslint.io/rules/prefer-nullish-coalescing/